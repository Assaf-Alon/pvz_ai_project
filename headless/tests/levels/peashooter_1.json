{
    "50": [["normal", 0]]
}
