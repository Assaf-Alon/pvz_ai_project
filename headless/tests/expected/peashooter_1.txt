[101] Sun autospawned. Total: 75.
[202] Sun autospawned. Total: 100.
[203] Planted peashooter in (0, 0). Total: 0.
[218] Pea generated by Peashooter in (0, 0).
[220] Pea in (0, 0) moved to (0, 1).
[222] Pea in (0, 1) moved to (0, 2).
[224] Pea in (0, 2) moved to (0, 3).
[226] Pea in (0, 3) moved to (0, 4).
[228] Pea in (0, 4) moved to (0, 5).
[230] Pea in (0, 5) moved to (0, 6).
[232] Pea in (0, 6) moved to (0, 7).
[233] Pea generated by Peashooter in (0, 0).
[234] Pea in (0, 7) moved to (0, 8).
[235] Pea in (0, 0) moved to (0, 1).
[236] Pea in (0, 8) moved to (0, 9).
[237] Pea in (0, 1) moved to (0, 2).
[238] Pea in (0, 9) moved to (0, 10).
[238] Pea in (0, 10) removed.
[239] Pea in (0, 2) moved to (0, 3).
[241] Pea in (0, 3) moved to (0, 4).
[243] Pea in (0, 4) moved to (0, 5).
[245] Pea in (0, 5) moved to (0, 6).
[247] Pea in (0, 6) moved to (0, 7).
[248] Pea generated by Peashooter in (0, 0).
[249] Pea in (0, 7) moved to (0, 8).
[250] Pea in (0, 0) moved to (0, 1).
[251] Pea in (0, 8) moved to (0, 9).
[252] Pea in (0, 1) moved to (0, 2).
[253] Pea in (0, 9) moved to (0, 10).
[253] Pea in (0, 10) removed.
[254] Pea in (0, 2) moved to (0, 3).
[256] Pea in (0, 3) moved to (0, 4).
[258] Pea in (0, 4) moved to (0, 5).
[260] Pea in (0, 5) moved to (0, 6).
[262] Pea in (0, 6) moved to (0, 7).
[263] Pea generated by Peashooter in (0, 0).
[264] Pea in (0, 7) moved to (0, 8).
[265] Pea in (0, 0) moved to (0, 1).
[266] Pea in (0, 8) moved to (0, 9).
[267] Pea in (0, 1) moved to (0, 2).
[268] Pea in (0, 9) moved to (0, 10).
[268] Pea in (0, 10) removed.
[269] Pea in (0, 2) moved to (0, 3).
[271] Pea in (0, 3) moved to (0, 4).
[273] Pea in (0, 4) moved to (0, 5).
[275] Pea in (0, 5) moved to (0, 6).
[277] Pea in (0, 6) moved to (0, 7).
[278] Pea generated by Peashooter in (0, 0).
[279] Pea in (0, 7) moved to (0, 8).
[280] Pea in (0, 0) moved to (0, 1).
[281] Pea in (0, 8) moved to (0, 9).
[282] Pea in (0, 1) moved to (0, 2).
[283] Pea in (0, 9) moved to (0, 10).
[283] Pea in (0, 10) removed.
[284] Pea in (0, 2) moved to (0, 3).
[286] Pea in (0, 3) moved to (0, 4).
[288] Pea in (0, 4) moved to (0, 5).
[290] Pea in (0, 5) moved to (0, 6).
[292] Pea in (0, 6) moved to (0, 7).
[293] Pea generated by Peashooter in (0, 0).
[294] Pea in (0, 7) moved to (0, 8).
[295] Pea in (0, 0) moved to (0, 1).
[296] Pea in (0, 8) moved to (0, 9).
[297] Pea in (0, 1) moved to (0, 2).
[298] Pea in (0, 9) moved to (0, 10).
[298] Pea in (0, 10) removed.
[299] Pea in (0, 2) moved to (0, 3).
[301] Pea in (0, 3) moved to (0, 4).
[303] Pea in (0, 4) moved to (0, 5).
[303] Sun autospawned. Total: 25.
[305] Pea in (0, 5) moved to (0, 6).
[307] Pea in (0, 6) moved to (0, 7).
[308] Pea generated by Peashooter in (0, 0).
[309] Pea in (0, 7) moved to (0, 8).
[310] Pea in (0, 0) moved to (0, 1).
[311] Pea in (0, 8) moved to (0, 9).
[312] Pea in (0, 1) moved to (0, 2).
[313] Pea in (0, 9) moved to (0, 10).
[313] Pea in (0, 10) removed.
[314] Pea in (0, 2) moved to (0, 3).
[316] Pea in (0, 3) moved to (0, 4).
[318] Pea in (0, 4) moved to (0, 5).
[320] Pea in (0, 5) moved to (0, 6).
[322] Pea in (0, 6) moved to (0, 7).
[323] Pea generated by Peashooter in (0, 0).
[324] Pea in (0, 7) moved to (0, 8).
[325] Pea in (0, 0) moved to (0, 1).
[326] Pea in (0, 8) moved to (0, 9).
[327] Pea in (0, 1) moved to (0, 2).
[328] Pea in (0, 9) moved to (0, 10).
[328] Pea in (0, 10) removed.
[329] Pea in (0, 2) moved to (0, 3).
[331] Pea in (0, 3) moved to (0, 4).
[333] Pea in (0, 4) moved to (0, 5).
[335] Pea in (0, 5) moved to (0, 6).
[337] Pea in (0, 6) moved to (0, 7).
[338] Pea generated by Peashooter in (0, 0).
[339] Pea in (0, 7) moved to (0, 8).
[340] Pea in (0, 0) moved to (0, 1).
[341] Pea in (0, 8) moved to (0, 9).
[342] Pea in (0, 1) moved to (0, 2).
[343] Pea in (0, 9) moved to (0, 10).
[343] Pea in (0, 10) removed.
[344] Pea in (0, 2) moved to (0, 3).
[346] Pea in (0, 3) moved to (0, 4).
[348] Pea in (0, 4) moved to (0, 5).
[350] Pea in (0, 5) moved to (0, 6).
[352] Pea in (0, 6) moved to (0, 7).
[353] Pea generated by Peashooter in (0, 0).
[354] Pea in (0, 7) moved to (0, 8).
[355] Pea in (0, 0) moved to (0, 1).
[356] Pea in (0, 8) moved to (0, 9).
[357] Pea in (0, 1) moved to (0, 2).
[358] Pea in (0, 9) moved to (0, 10).
[358] Pea in (0, 10) removed.
[359] Pea in (0, 2) moved to (0, 3).
[361] Pea in (0, 3) moved to (0, 4).
[363] Pea in (0, 4) moved to (0, 5).
[365] Pea in (0, 5) moved to (0, 6).
[367] Pea in (0, 6) moved to (0, 7).
[368] Pea generated by Peashooter in (0, 0).
[369] Pea in (0, 7) moved to (0, 8).
[370] Pea in (0, 0) moved to (0, 1).
[371] Pea in (0, 8) moved to (0, 9).
[372] Pea in (0, 1) moved to (0, 2).
[373] Pea in (0, 9) moved to (0, 10).
[373] Pea in (0, 10) removed.
[374] Pea in (0, 2) moved to (0, 3).
[376] Pea in (0, 3) moved to (0, 4).
[378] Pea in (0, 4) moved to (0, 5).
[380] Pea in (0, 5) moved to (0, 6).
[382] Pea in (0, 6) moved to (0, 7).
[383] Pea generated by Peashooter in (0, 0).
[384] Pea in (0, 7) moved to (0, 8).
[385] Pea in (0, 0) moved to (0, 1).
[386] Pea in (0, 8) moved to (0, 9).
[387] Pea in (0, 1) moved to (0, 2).
[388] Pea in (0, 9) moved to (0, 10).
[388] Pea in (0, 10) removed.
[389] Pea in (0, 2) moved to (0, 3).
[391] Pea in (0, 3) moved to (0, 4).
[393] Pea in (0, 4) moved to (0, 5).
[395] Pea in (0, 5) moved to (0, 6).
[397] Pea in (0, 6) moved to (0, 7).
[398] Pea generated by Peashooter in (0, 0).
[399] Pea in (0, 7) moved to (0, 8).
[400] Pea in (0, 0) moved to (0, 1).
[401] Pea in (0, 8) moved to (0, 9).
[402] Pea in (0, 1) moved to (0, 2).
[403] Pea in (0, 9) moved to (0, 10).
[403] Pea in (0, 10) removed.
[404] Pea in (0, 2) moved to (0, 3).
[404] Sun autospawned. Total: 50.
[406] Pea in (0, 3) moved to (0, 4).
[408] Pea in (0, 4) moved to (0, 5).
[410] Pea in (0, 5) moved to (0, 6).
[412] Pea in (0, 6) moved to (0, 7).
[413] Pea generated by Peashooter in (0, 0).
[414] Pea in (0, 7) moved to (0, 8).
[415] Pea in (0, 0) moved to (0, 1).
[416] Pea in (0, 8) moved to (0, 9).
[417] Pea in (0, 1) moved to (0, 2).
[418] Pea in (0, 9) moved to (0, 10).
[418] Pea in (0, 10) removed.
[419] Pea in (0, 2) moved to (0, 3).
[421] Pea in (0, 3) moved to (0, 4).
[423] Pea in (0, 4) moved to (0, 5).
[425] Pea in (0, 5) moved to (0, 6).
[427] Pea in (0, 6) moved to (0, 7).
[428] Pea generated by Peashooter in (0, 0).
[429] Pea in (0, 7) moved to (0, 8).
[430] Pea in (0, 0) moved to (0, 1).
[431] Pea in (0, 8) moved to (0, 9).
[432] Pea in (0, 1) moved to (0, 2).
[433] Pea in (0, 9) moved to (0, 10).
[433] Pea in (0, 10) removed.
[434] Pea in (0, 2) moved to (0, 3).
[436] Pea in (0, 3) moved to (0, 4).
[438] Pea in (0, 4) moved to (0, 5).
[440] Pea in (0, 5) moved to (0, 6).
[442] Pea in (0, 6) moved to (0, 7).
[443] Pea generated by Peashooter in (0, 0).
[444] Pea in (0, 7) moved to (0, 8).
[445] Pea in (0, 0) moved to (0, 1).
[446] Pea in (0, 8) moved to (0, 9).
[447] Pea in (0, 1) moved to (0, 2).
[448] Pea in (0, 9) moved to (0, 10).
[448] Pea in (0, 10) removed.
[449] Pea in (0, 2) moved to (0, 3).
[451] Pea in (0, 3) moved to (0, 4).
[453] Pea in (0, 4) moved to (0, 5).
[455] Pea in (0, 5) moved to (0, 6).
[457] Pea in (0, 6) moved to (0, 7).
[458] Pea generated by Peashooter in (0, 0).
[459] Pea in (0, 7) moved to (0, 8).
[460] Pea in (0, 0) moved to (0, 1).
[461] Pea in (0, 8) moved to (0, 9).
[462] Pea in (0, 1) moved to (0, 2).
[463] Pea in (0, 9) moved to (0, 10).
[463] Pea in (0, 10) removed.
[464] Pea in (0, 2) moved to (0, 3).
[466] Pea in (0, 3) moved to (0, 4).
[468] Pea in (0, 4) moved to (0, 5).
[470] Pea in (0, 5) moved to (0, 6).
[472] Pea in (0, 6) moved to (0, 7).
[473] Pea generated by Peashooter in (0, 0).
[474] Pea in (0, 7) moved to (0, 8).
[475] Pea in (0, 0) moved to (0, 1).
[476] Pea in (0, 8) moved to (0, 9).
[477] Pea in (0, 1) moved to (0, 2).
[478] Pea in (0, 9) moved to (0, 10).
[478] Pea in (0, 10) removed.
[479] Pea in (0, 2) moved to (0, 3).
[481] Pea in (0, 3) moved to (0, 4).
[483] Pea in (0, 4) moved to (0, 5).
[485] Pea in (0, 5) moved to (0, 6).
[487] Pea in (0, 6) moved to (0, 7).
[488] Pea generated by Peashooter in (0, 0).
[489] Pea in (0, 7) moved to (0, 8).
[490] Pea in (0, 0) moved to (0, 1).
[491] Pea in (0, 8) moved to (0, 9).
[492] Pea in (0, 1) moved to (0, 2).
[493] Pea in (0, 9) moved to (0, 10).
[493] Pea in (0, 10) removed.
[494] Pea in (0, 2) moved to (0, 3).
[496] Pea in (0, 3) moved to (0, 4).
[498] Pea in (0, 4) moved to (0, 5).
[500] Pea in (0, 5) moved to (0, 6).
[500] Zombie being spawned at lane 0
[502] Pea in (0, 6) moved to (0, 7).
[503] Pea generated by Peashooter in (0, 0).
[504] Pea in (0, 7) moved to (0, 8).
[505] Pea in (0, 0) moved to (0, 1).
[505] Sun autospawned. Total: 75.
[506] Pea in (0, 8) moved to (0, 9).
[506] Pea in (0, 9) attacked.
[506] Zombie in (0, 9) was damaged. HP: 161.
[506] Pea in (0, 9) removed.
[507] Pea in (0, 1) moved to (0, 2).
[509] Pea in (0, 2) moved to (0, 3).
[511] Pea in (0, 3) moved to (0, 4).
[513] Pea in (0, 4) moved to (0, 5).
[515] Pea in (0, 5) moved to (0, 6).
[517] Pea in (0, 6) moved to (0, 7).
[518] Pea generated by Peashooter in (0, 0).
[519] Pea in (0, 7) moved to (0, 8).
[520] Pea in (0, 0) moved to (0, 1).
[521] Pea in (0, 8) moved to (0, 9).
[521] Pea in (0, 9) attacked.
[521] Zombie in (0, 9) was damaged. HP: 141.
[521] Pea in (0, 9) removed.
[522] Pea in (0, 1) moved to (0, 2).
[524] Pea in (0, 2) moved to (0, 3).
[526] Pea in (0, 3) moved to (0, 4).
[528] Pea in (0, 4) moved to (0, 5).
[530] Pea in (0, 5) moved to (0, 6).
[532] Pea in (0, 6) moved to (0, 7).
[533] Pea generated by Peashooter in (0, 0).
[534] Pea in (0, 7) moved to (0, 8).
[535] Pea in (0, 0) moved to (0, 1).
[536] Pea in (0, 8) moved to (0, 9).
[536] Pea in (0, 9) attacked.
[536] Zombie in (0, 9) was damaged. HP: 121.
[536] Pea in (0, 9) removed.
[537] Pea in (0, 1) moved to (0, 2).
[539] Pea in (0, 2) moved to (0, 3).
[541] Pea in (0, 3) moved to (0, 4).
[543] Pea in (0, 4) moved to (0, 5).
[545] Pea in (0, 5) moved to (0, 6).
[547] Pea in (0, 6) moved to (0, 7).
[547] Zombie in (0, 9) moved to (0, 8).
[548] Pea generated by Peashooter in (0, 0).
[549] Pea in (0, 7) moved to (0, 8).
[549] Pea in (0, 8) attacked.
[549] Zombie in (0, 8) was damaged. HP: 101.
[549] Pea in (0, 8) removed.
[550] Pea in (0, 0) moved to (0, 1).
[552] Pea in (0, 1) moved to (0, 2).
[554] Pea in (0, 2) moved to (0, 3).
[556] Pea in (0, 3) moved to (0, 4).
[558] Pea in (0, 4) moved to (0, 5).
[560] Pea in (0, 5) moved to (0, 6).
[562] Pea in (0, 6) moved to (0, 7).
[563] Pea generated by Peashooter in (0, 0).
[564] Pea in (0, 7) moved to (0, 8).
[564] Pea in (0, 8) attacked.
[564] Zombie in (0, 8) was damaged. HP: 81.
[564] Pea in (0, 8) removed.
[565] Pea in (0, 0) moved to (0, 1).
[567] Pea in (0, 1) moved to (0, 2).
[569] Pea in (0, 2) moved to (0, 3).
[571] Pea in (0, 3) moved to (0, 4).
[573] Pea in (0, 4) moved to (0, 5).
[575] Pea in (0, 5) moved to (0, 6).
[577] Pea in (0, 6) moved to (0, 7).
[578] Pea generated by Peashooter in (0, 0).
[579] Pea in (0, 7) moved to (0, 8).
[579] Pea in (0, 8) attacked.
[579] Zombie in (0, 8) was damaged. HP: 61.
[579] Pea in (0, 8) removed.
[580] Pea in (0, 0) moved to (0, 1).
[582] Pea in (0, 1) moved to (0, 2).
[584] Pea in (0, 2) moved to (0, 3).
[586] Pea in (0, 3) moved to (0, 4).
[588] Pea in (0, 4) moved to (0, 5).
[590] Pea in (0, 5) moved to (0, 6).
[592] Pea in (0, 6) moved to (0, 7).
[593] Pea generated by Peashooter in (0, 0).
[594] Pea in (0, 7) moved to (0, 8).
[594] Pea in (0, 8) attacked.
[594] Zombie in (0, 8) was damaged. HP: 41.
[594] Pea in (0, 8) removed.
[594] Zombie in (0, 8) moved to (0, 7).
[595] Pea in (0, 0) moved to (0, 1).
[597] Pea in (0, 1) moved to (0, 2).
[599] Pea in (0, 2) moved to (0, 3).
[601] Pea in (0, 3) moved to (0, 4).
[603] Pea in (0, 4) moved to (0, 5).
[605] Pea in (0, 5) moved to (0, 6).
[606] Sun autospawned. Total: 100.
[607] Pea in (0, 6) moved to (0, 7).
[607] Pea in (0, 7) attacked.
[607] Zombie in (0, 7) was damaged. HP: 21.
[607] Pea in (0, 7) removed.
[607] Planted peashooter in (1, 0). Total: 0.
[608] Pea generated by Peashooter in (0, 0).
[610] Pea in (0, 0) moved to (0, 1).
[612] Pea in (0, 1) moved to (0, 2).
[614] Pea in (0, 2) moved to (0, 3).
[616] Pea in (0, 3) moved to (0, 4).
[618] Pea in (0, 4) moved to (0, 5).
[620] Pea in (0, 5) moved to (0, 6).
[622] Pea generated by Peashooter in (1, 0).
[622] Pea in (0, 6) moved to (0, 7).
[622] Pea in (0, 7) attacked.
[622] Zombie in (0, 7) was damaged. HP: 1.
[622] Pea in (0, 7) removed.
[623] Pea generated by Peashooter in (0, 0).
[624] Pea in (1, 0) moved to (1, 1).
[625] Pea in (0, 0) moved to (0, 1).
[626] Pea in (1, 1) moved to (1, 2).
[627] Pea in (0, 1) moved to (0, 2).
[628] Pea in (1, 2) moved to (1, 3).
[629] Pea in (0, 2) moved to (0, 3).
[630] Pea in (1, 3) moved to (1, 4).
[631] Pea in (0, 3) moved to (0, 4).
[632] Pea in (1, 4) moved to (1, 5).
[633] Pea in (0, 4) moved to (0, 5).
[634] Pea in (1, 5) moved to (1, 6).
[635] Pea in (0, 5) moved to (0, 6).
[636] Pea in (1, 6) moved to (1, 7).
[637] Pea generated by Peashooter in (1, 0).
[637] Pea in (0, 6) moved to (0, 7).
[637] Pea in (0, 7) attacked.
[637] Zombie in (0, 7) was damaged. HP: -19.
[637] Zombie in (0, 7) was killed.
[637] Pea in (0, 7) removed.
[637] You've won!
